app/tests/unit/containers/Products/productRowItemContainer.test.tsx

```diff
@@ -174,19 +176,30 @@ describe('ProductList', () => {
 
   it('should not render the benchmark setting dropdown option when the product is not a ciip-benchmarked product', async () => {
     const testProduct = {...product, isCiipProduct: false};
-    const r = mount(
-      <table>
-        <tbody>
-          <ProductRowItemComponent product={testProduct} query={query} />
-        </tbody>
-      </table>
-    );
-    r.find('DropdownToggle').simulate('click');
+
+    let rendered;
+
+    await act(async () => {
+      rendered = await mount(
+        <table>
+          <tbody>
+            <ProductRowItemComponent product={testProduct} query={query} />
+          </tbody>
+        </table>
+      );
+      await rendered.find('DropdownToggle').simulate('click');
+    });
```

Comment:
this is the only thing that needed more attention, according to React docs, anything that modifies the DOM should be wrapped in `act(() => { ... })`. I wonder why it's only complaining here?

Reply:
huh, not sure. maybe a simulate directly off a find?

Reply:
all the other tests in this file also do that, they just don't try to test the dropdown while it's open. There must be some particulars in the implementation from bootstrap